xml/cl.xml

```diff
@@ -1787,8 +1787,13 @@ server's OpenCL/api-docs repository.
             <unused start="0x4214" end="0x421F"/>
     </enums>
 
-    <enums start="0x4220" end="0x4FFF" name="enums.4220" comment="Reserved for vendor extensions. Allocate in groups of 16.">
-            <unused start="0x420F" end="0x4FFF"/>
+    <enums start="0x4220" end="0x422F" name="enums.4220" comment="Reserved for EXT extensions">
+        <enum value="0x4220"        name="CL_DEVICE_CXX_FOR_OPENCL_VERSION_EXT"/>
```

Comment:
I've just realized that I merged an XML change that reserves this block: https://github.com/KhronosGroup/OpenCL-Docs/pull/411

Hopefully it's not an issue to change this number?  Sorry I didn't make the connection despite reviewing both PRs.  If there are concerns about it happening again before this branch merges then the XML change could be its own PR.

Reply:
I've changed the allocation. This should be fine.